erfa/tests/helper.py

```diff
@@ -80,7 +80,7 @@ def treat_deprecations_as_exceptions():
     except ImportError:
         pass
     else:
-        if LooseVersion(matplotlib.__version__) < '3':
```

Comment:
I wonder if we should just go for `matplotlib.__version__[0] < '3'` and avoid using `packaging` at all?

Or, indeed, shall we just remove this? 3.0 appeared in 2018!

Reply:
OK I went for the first option, just to stay on the safe side ...